Joining together CTRP data.Rmd

```diff
@@ -0,0 +1,304 @@
+
+vorinostat1 <- final_data %>%
+    select("Cell_line_name", "Cell_line_ID", "Cancer_type", "Compound_name", "Compound_ID", "AUC") %>%
+    separate(col = "Compound_name", into = c("Compound_1", "Compound_2")) %>%
+    filter(Compound_1 == "vorinostat" | Compound_2 == "vorinostat")
+
+ggplot(vorinostat1, aes(x = Cancer_type, y = AUC, fill = Cancer_type)) +
+
+    geom_boxplot() +
+
+    labs(title = "AUC values of different cancer types to vorinostat alone \nor in combination with other drug", x = "Cancer type", y = "AUC") +
+
+    theme(plot.title = element_text(size = 14, family = "Tahoma", face = "bold", hjust = 0.5),
+          plot.title.position = "plot",
+          legend.position = "none",
+          axis.title.x = element_text(colour="black", size = 12, face = "bold"),
+          axis.title.y = element_text(colour="black", size = 12, face = "bold"),
+          axis.ticks.x = element_line(size = 0.5),
+          axis.text.y = element_text(colour="black", size = 10),
+          panel.grid.major.x = element_blank(),
+          panel.background = element_rect(fill = "white", color = "black")) +
+    
+    scale_y_continuous(breaks = c(0, 2, 4, 6, 8, 10, 12, 14, 16, 18, 20), limits = c(0, 20)) +
+    coord_flip()
+
+# Calculate mean AUC for vorinostat used alone
+
+vorinostat2_mean <- final_data %>%
+    select("Cell_line_name", "Cell_line_ID", "Cancer_type", "Compound_name", "Compound_ID", "AUC") %>%
+    separate(col = "Compound_name", into = c("Compound_1", "Compound_2")) %>%
+    filter(Compound_ID == 56554) %>%
+    group_by(Cancer_type) %>%
+    summarize(AUC = round(mean(AUC), digits = 2)) %>%
+    arrange(AUC)
+
+table9 <- head(vorinostat2_mean)
+kable(table9, align = "lc", col.names = c("Cancer type", "Mean AUC"))
+
+# Plot mean AUC for vorinostat used alone
+
+vorinostat2 <- final_data %>%
+    select("Cell_line_name", "Cell_line_ID", "Cancer_type", "Compound_name", "Compound_ID", "AUC") %>%
+    separate(col = "Compound_name", into = c("Compound_1", "Compound_2")) %>%
+    filter(Compound_ID == 56554)
+
+ggplot(vorinostat2, aes(x = Cancer_type, y = AUC, fill = Cancer_type)) +
+
+    geom_boxplot() +
+
+    labs(title = "AUC values of different cancer types to vorinostat alone", x = "Cancer type", y = "AUC") +
+
+    theme(plot.title = element_text(size = 14, family = "Tahoma", face = "bold", hjust = 0.5),
+          plot.title.position = "plot",
+          legend.position = "none",
+          axis.title.x = element_text(colour="black", size = 12, face = "bold"),
+          axis.title.y = element_text(colour="black", size = 12, face = "bold"),
+          axis.ticks.x = element_line(size = 0.5),
+          axis.text.y = element_text(colour="black", size = 10),
+          panel.grid.major.x = element_blank(),
+          panel.background = element_rect(fill = "white", color = "black")) +
+    
+    scale_y_continuous(breaks = c(0, 2, 4, 6, 8, 10, 12, 14, 16, 18, 20), limits = c(0, 20)) +
+    coord_flip()
```

Comment:
It's a great idea to show the mean values in a table in addition to the boxplot. Although the boxplot looks very nice and clean, it's not immediately evident which cancer type has the lowest AUC values. The table definitely helps with clarification. Nice!